src/templates/staff/assignment.html

```diff
@@ -101,7 +101,7 @@
                             + '</td><td class="text-right">' + fmtTimestamp(result[i].end)
                             + '</td><td class="text-right">' + result[i].max_runs
                             + '</td><td class="text-right">' + result[i].remaining_runs
-                            + '</td><td class="text-right"><button id="btn-' + result[i]._id + '" class="btn btn-danger" onclick="deleteExtension(\'' + result[i]._id + '\')">Delete</button>'
+                            + '</td><td><button id="btn-' + result[i]._id + '" class="btn btn-outline-danger fas fa-trash-alt" style="cursor: pointer" onclick="deleteExtension(\'' + result[i]._id + '\')"/>'
```

Comment:
I actually meant replacing the whole button with an icon 😅, something similar to the roster deletion button. Sorry for not being clear